### ISSUE TITLE
Python Web Scrapper and Email Automated delivery

### ISSUE THREAD

**Issue by ShivamPathak99** [python-geeks/Automation-scripts#890]
## Python Web Scrapper and Automated Email Delivery System

**Brief** - This is a python automated Web Scrapper which pulls the list of headlines from HackerNews webpage and list them, after that its sends the list with their link through email to other people.

## Instructions

- Create a new folder for your script and file/folder name should be appropriate.
- Create a `README.md` (**[using this template](https://github.com/python-geeks/Automation-scripts/blob/main/README_TEMPLATE.md)**) in your folder for program Instructions
- add requirements.txt if needed

### Programming Language

- [x] Python

Happy Coding

**Comment by royninja** [comment]
Hi I would like to know how you want to pass the email? By writing each and every email on the terminal?

**Comment by ShivamPathak99** [comment]
No, as of now I haven't added any input for email, email is written in code. But, I was going to implement it by taking emails from Excel sheets.

**Comment by Rozaki0318** [comment]
I'm willing to contribute for this issue.

**Comment by ur0dead** [comment]
Hi! I’d like to work on this issue. Can you please assign it to me?
